src/site/content/en/blog/display-override/index.md

```diff
@@ -0,0 +1,135 @@
+As a quick recap, below are the display modes that are currently specified.
+
+<div class="w-table-wrapper">
+  <table id="display-params">
+    <thead>
+      <tr>
+        <th><strong>Property</strong></th>
+        <th><strong>Use</strong></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>fullscreen</code></td>
+        <td>
+          Opens the web application without any browser UI and takes
+          up the entirety of the available display area.
+        </td>
+      </tr>
+      <tr>
+        <td><code>standalone</code></td>
+        <td>
+          Opens the web app to look and feel like a standalone
+          app. The app runs in its own window, separate from the browser, and
+          hides standard browser UI elements like the URL bar.
+        </td>
+      </tr>
+      <tr>
+        <td><code>minimal-ui</code></td>
+        <td>
+          This mode is similar to <code>standalone</code>, but provides the
+          user a minimal set of UI elements for controlling navigation (such
+          as back and reload).
+        </td>
+      </tr>
+      <tr>
+        <td><code>browser</code></td>
+        <td>A standard browser experience.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+These display modes follow a [well-defined fallback chain](https://w3c.github.io/manifest/#dfn-fallback-display-mode)
+(`"fullscreen"` → `"standalone"` → `"minimal-ui"` → `"browser"`). If a browser does not support a given
+mode, it falls back to the next display mode in the chain.
+
+## Shortcomings of the `display` property
+
+The problem with this hard-wired fallback chain approach is threefold:
+
+- A developer cannot request `"minimal-ui"` without being forced back into the `"browser"` display mode in case `"minimal-ui"` is not supported by a given browser.
+- Developers have no way of handling cross-browser differences, like if the browser includes or excludes a [back button](https://twitter.com/ChromiumDev/status/1012065260625383425/photo/1) in the window for `"standalone"` mode.
+- The current behavior makes it impossible to introduce new display
+modes in a backward compatible way, since explorations like tabbed application mode do not have a
+natural place in the fallback chain.
+
+## The `display_override` property
+
+{% Aside %}
+  The `display_override` property is meant to solve special corner cases. In almost all
+  circumstances the regular `display` property is what developers should reach out for.
+{% endAside %}
+
+These problems are solved by the `display_override` property, which the browser considers *before*
+the `display` property. Its value is a sequence of strings that are considered in-order, and the
+first supported display mode is applied. If none are supported, the browser falls back to evaluating
+the `display` field.
+
+In the example below, the display mode fallback chain would be as follows.
+(The details of `"window-control-overlay"` are out-of-scope for this article.)
```

Comment:
That seems strange since it's one of the possible values for `display_override`.

Reply:
But one of potentially many. In the concrete case, I have an article coming at https://github.com/GoogleChrome/web.dev/pull/4680 that we can link to once it's live.